anvil-python/anvil/provider/local/commands.py

```diff
@@ -117,6 +118,78 @@ def remove_trailing_dot(value: str) -> str:
     return value.rstrip(".")
 
 
+def postgresql_steps(
+    client: Client,
+    manifest: Manifest,
+    jhelper: JujuHelper,
+    deployment: LocalDeployment,
+    fqdn: str,
+) -> List[BaseStep]:
+    return [
+        TerraformInitStep(manifest.get_tfhelper("postgresql-plan")),
+        DeployPostgreSQLApplicationStep(
+            client, manifest, jhelper, deployment.infrastructure_model
+        ),
+        AddPostgreSQLUnitsStep(
+            client, fqdn, jhelper, deployment.infrastructure_model
+        ),
+    ]
```

Comment:
I'm wondering whether this shouldn't be defined in `anvil.commands.postgresql`
and then change the import above to 
`from anvil.commands.postgresql import postgresql_steps`

Reply:
(I mean the entire postgresql_steps block, not just this part of the list)